src/main/resources/config/application-dev.yml

```diff
@@ -58,10 +58,19 @@ spring:
     liquibase:
         contexts: dev
     mail:
-        host: localhost
-        port: 25
-        username:
-        password:
+        host: smtp.gmail.com
+        port: 587
+        username: snorlax.io2019@gmail.com  # Replace this field with your Gmail username.
```

Comment:
Also: could you create a snorlax email? No one will remember about it before the demo :)